docs/section22/Rule22-7.md

```diff
@@ -0,0 +1,56 @@
+
+# CHW&CW - Rule 22-7  
+
+**Rule ID:** 22-7  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water shall be modeled as primary/secondary systems.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is not modeled with purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is not modeled with purchased chilled water: `if Rule-18-8 == "NOT APPLICABLE":`
+
+## Rule Logic:  
+
+- For each building segment in B_RMR: `for building_segment_b in B_RMR...building_segments:`
+
+  - For each HVAC system in building segment: `for hvac_b in building_segment_b.heating_ventilation_air_conditioning_systems`
+
+    - Check if HVAC system has chilled water coil: `if hvac_b.cooling_system.chilled_water_loop:`
+
```

Comment:
I think we also need to check that this loop is not referenced in any of the chller.cooling_loop and is referenced in FluidLoop.child_loops

Reply:
small correct to line 30, cooling system would be a "fluid loop" and there needs to be an additional check that verifies that the fluid loop type = "cooling"